docs/core/tools/dependencies.md

```diff
@@ -1,66 +1,72 @@
 ```
 
-The dependency will only be valid if the build is happening for that given target. The `$(TargetFramework)` in the condition is an MSBuild property that's being set in the project. For most common .NET Core applications, you will not need to do this.
-
-## Add a dependency to the project
-
-Adding a dependency to your project is straightforward. Here is an example of how to add Json.NET version `9.0.1` to your project. Of course, it is applicable to any other NuGet dependency.
+The dependency in the preceding example will only be valid if the build is happening for that given target. The `$(TargetFramework)` in the condition is an MSBuild property that's being set in the project. For most common .NET Core applications, you don't need to do this.
 
-Your project file has two or more `<ItemGroup>` nodes. One of the nodes already has `<PackageReference>` elements in it. You can add your new dependency to this node or create a new one; the result will be the same.
+## Add a dependency by editing the project file
 
-The following example uses the default template that's dropped by `dotnet new console`. This is a simple console application. When you open up the project, you'll find the `<ItemGroup>` with already existing `<PackageReference>` in it. Add the following to it:
+To add a dependency, add a `<PackageReference>` element inside an `<ItemGroup>` element. You can add to an existing `<ItemGroup>` or create a new one. The following example uses the default console application project that's created by `dotnet new console`:
 
 ```xml
-<PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-```
-
-After this, save the project and run the `dotnet restore` command to install the dependency.
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
-
-The full project looks like this:
-
-```xml
-<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.2" />
   </ItemGroup>
+
 </Project>
 ```
 
-## Remove a dependency from the project
+## Add a dependency by using the CLI
+
+To add a dependency, run the [dotnet add package](dotnet-add-package.md) command, as shown in the following example:
+
+```dotnetcli
+dotnet add package Microsoft.EntityFrameworkCore
+```
+
+## Remove a dependency by editing the project file
+
+To remove a dependency, remove its `<PackageReference>` element from the project file.
+
+## Remove a dependency by using the CLI
+
+To remove a dependency, run the [dotnet remove package](dotnet-remove-package.md) command, as shown in the following example:
```

Comment:
Just thinking about this, won't the **dotnet remove package** get localized because it's not in `code` syntax? Maybe need to add the no-loc tags for this and other commands in this article.

Reply:
Good catch, thanks!